server/routes/api/place.py

```diff
@@ -192,23 +192,23 @@ def statsvars_route(dcid):
     Returns:
       A list of statistical variable dcids.
     """
-    return Response(json.dumps(statsvars(dcid)),
+    return Response(json.dumps(statVars(dcid)),
                     200,
                     mimetype='application/json')
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def statsvars(dcid):
+def statVars(dcid):
```

Comment:
looking at the other file you changed, maybe for consistency we should do stat_vars?

Reply:
Updated